proposals/TP1-xds-transport-next.md

```diff
@@ -0,0 +1,1082 @@
+xdstp:// structured resource naming, caching and federation support.
```

Comment:
Minor nit: Please add "TP1:" at the start of this line, so the xRFC number is clear to readers.

Might also be a good idea to add the xRFC number to the PR description, too.